hypothesis-python/tests/django/toystore/models.py

```diff
@@ -47,7 +47,7 @@ class Customish(models.Model):
 class Customer(models.Model):
     name = models.CharField(max_length=100, unique=True)
     email = models.EmailField(max_length=100, unique=True)
-    gender = models.CharField(max_length=50, null=True)
+    gender = models.CharField(max_length=50, null=True, blank=True)
```

Comment:
Why has this changed?

Reply:
Oh, I was trying to get the model form working right. I can revert it now.